hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/AbiConstants.java

```diff
@@ -1,119 +1,120 @@
+    // transferNFTs(address token, address[] memory sender, address[] memory receiver, int64[]
+    // memory serialNumber)
+    public static final int ABI_ID_TRANSFER_NFTS = 0x2c4ba191;
+    // transferNFT(address token,  address sender, address recipient, int64 serialNum)
+    public static final int ABI_ID_TRANSFER_NFT = 0x5cfc9011;
+    // mintToken(address token, uint64 amount, bytes[] memory metadata)
+    public static final int ABI_ID_MINT_TOKEN = 0x278e0b88;
+    // burnToken(address token, uint64 amount, int64[] memory serialNumbers)
+    public static final int ABI_ID_BURN_TOKEN = 0xacb9cff9;
+    // associateTokens(address account, address[] memory tokens)
+    public static final int ABI_ID_ASSOCIATE_TOKENS = 0x2e63879b;
+    // associateToken(address account, address token)
+    public static final int ABI_ID_ASSOCIATE_TOKEN = 0x49146bde;
+    // dissociateTokens(address account, address[] memory tokens)
+    public static final int ABI_ID_DISSOCIATE_TOKENS = 0x78b63918;
+    // dissociateToken(address account, address token)
+    public static final int ABI_ID_DISSOCIATE_TOKEN = 0x099794e8;
+
+    // **** HIP-218 + HIP-376 function selectors and event signatures ****
+    // redirectForToken(address token, bytes memory data)
+    public static final int ABI_ID_REDIRECT_FOR_TOKEN = 0x618dc65e;
+    // name()
+    public static final int ABI_ID_ERC_NAME = 0x06fdde03;
+    // symbol()
+    public static final int ABI_ID_ERC_SYMBOL = 0x95d89b41;
+    // decimals()
+    public static final int ABI_ID_ERC_DECIMALS = 0x313ce567;
+    // totalSupply()
+    public static final int ABI_ID_ERC_TOTAL_SUPPLY_TOKEN = 0x18160ddd;
+    // balanceOf(address account)
+    public static final int ABI_ID_ERC_BALANCE_OF_TOKEN = 0x70a08231;
+    // transfer(address recipient, uint256 amount)
+    public static final int ABI_ID_ERC_TRANSFER = 0xa9059cbb;
+    // transferFrom(address sender, address recipient, uint256 amount)
+    // transferFrom(address from, address to, uint256 tokenId)
+    public static final int ABI_ID_ERC_TRANSFER_FROM = 0x23b872dd;
+    // allowance(address owner, address spender)
+    public static final int ABI_ID_ERC_ALLOWANCE = 0xdd62ed3e;
+    // approve(address spender, uint256 amount)
+    // approve(address to, uint256 tokenId)
+    public static final int ABI_ID_ERC_APPROVE = 0x95ea7b3;
+    // setApprovalForAll(address operator, bool approved)
+    public static final int ABI_ID_ERC_SET_APPROVAL_FOR_ALL = 0xa22cb465;
+    // getApproved(uint256 tokenId)
+    public static final int ABI_ID_ERC_GET_APPROVED = 0x081812fc;
+    // isApprovedForAll(address owner, address operator)
+    public static final int ABI_ID_ERC_IS_APPROVED_FOR_ALL = 0xe985e9c5;
+    // ownerOf(uint256 tokenId)
+    public static final int ABI_ID_ERC_OWNER_OF_NFT = 0x6352211e;
+    // tokenURI(uint256 tokenId)
+    public static final int ABI_ID_ERC_TOKEN_URI_NFT = 0xc87b56dd;
+
+    // pauseToken(address)
```

Comment:
To be consistent with the other comments you can add argument name : `pauseToken(address token)`